core/src/commonMain/kotlin/com/ustadmobile/core/controller/ContentEntryEdit2Presenter.kt

```diff
@@ -208,6 +216,10 @@ class ContentEntryEdit2Presenter(context: Any,
                 if (entryUid != null) entry.contentEntryUid = entryUid.toString().toLong()
                 view.fileImportErrorVisible = false
                 view.entity = entry
+                if(metadata.mimeType == "video/mp4"){
```

Comment:
Why only mp4? Why not just anything that starts with "video/"? This would exclude webm etc.

Reply:
done